recipes-nodes/rbuilder/rbuilder.inc

```diff
@@ -109,24 +109,24 @@ EXTRA_OECARGO_BUILDARGS += "--features modern --frozen"
 
 do_install:append() {
     install -d ${D}${sysconfdir}/init.d
+    install -d ${D}/persistent/rbuilder
```

Comment:
Not sure how this will be installed because upon boot, the disk-encryption will format that path and mount it as /persistent.
So this might cause some runtime issues